eos/form-factors/parametric-kkvdz2022.cc

```diff
@@ -0,0 +1,175 @@
+                    UsedParameter(p[par_name("omega_4_1")],  *this),
+                    UsedParameter(p[par_name("omega_4_2")],  *this) }},
+        a_rho_1{{ UsedParameter(p[par_name("rho_1_0")],  *this),
+                  UsedParameter(p[par_name("rho_1_1")],  *this),
+                  UsedParameter(p[par_name("rho_1_2")],  *this) }},
+        a_rho_2{{ UsedParameter(p[par_name("rho_2_0")],  *this),
+                  UsedParameter(p[par_name("rho_2_1")],  *this),
+                  UsedParameter(p[par_name("rho_2_2")],  *this) }},
+        a_rho_3{{ UsedParameter(p[par_name("rho_3_0")],  *this),
+                  UsedParameter(p[par_name("rho_3_1")],  *this),
+                  UsedParameter(p[par_name("rho_3_2")],  *this) }},
+        a_rho_4{{ UsedParameter(p[par_name("rho_4_0")],  *this),
+                  UsedParameter(p[par_name("rho_4_1")],  *this),
+                  UsedParameter(p[par_name("rho_4_2")],  *this) }}
+    {
+    }
+
+    FormFactors<PToGammaOffShell> *
+    KKvDZ2022FormFactors::make(const Parameters & p, const Options & o)
+    {
+        return new KKvDZ2022FormFactors(p, o);
+    }
+
+    std::string
+    KKvDZ2022FormFactors::par_name(const std::string & ff_name)
+    {
+        return std::string("B->gamma^*") + std::string("::N^") + ff_name + std::string("@KKvDZ2022");
+    }
+
+    double
+    KKvDZ2022FormFactors::_widthOmega(const double & q2) const
+    {
+        const double constant_omega_width_approximation = 0.00868;
+
+        if (q2 - 9.0 * m_PionSq > 0.0)
+        {
+            return constant_omega_width_approximation;
+        }
+        if (q2 - 9.0 * m_PionSq <= 0.0)
+        {
+            return 0.0;
+        }
+    }
+    
+    double
+    KKvDZ2022FormFactors::_widthRho(const double & q2) const
+    {
+        const double constant_rho_width_approximation = 0.1474;
+
+        if (q2 - 4.0 * m_PionSq > 0.0)
+        {
+            return std::pow((q2 - 4.0 * m_PionSq)/(m_RhoSq - 4.0 * m_PionSq), 1.5) * m_RhoSq/q2 * constant_rho_width_approximation;
+        }
+        if (q2 - 4.0 * m_PionSq <= 0.0)
+        {
+            return 0.0;
+        }
+    }
+
+    double
+    KKvDZ2022FormFactors::_zOmega(const double & k2) const
+    {
+        return (sqrt(tPOmega - k2) - sqrt(tPOmega - t0Omega))/(sqrt(tPOmega - k2) + sqrt(tPOmega - t0Omega));
+    }
+
+    double
+    KKvDZ2022FormFactors::_zRho(const double & k2) const
+    {
+        return (sqrt(tPRho - k2) - sqrt(tPRho - t0Rho))/(sqrt(tPRho - k2) + sqrt(tPRho - t0Rho));
+    }
+
+    complex<double>
+    KKvDZ2022FormFactors::_calc_ff_contribution_omega(const double & q2, const double & k2, const double & m2_R, const std::array<UsedParameter, 3> & a) const
+    {
+        double a_0(a[0]),a_1(a[1]),a_2(a[2]);
+
+        const double diff_z = _zOmega(k2) - _zOmega(0.0);
+        return 1.0 / (1.0 - k2 / m2_R) * ( m_OmegaSq / ( m_OmegaSq - q2 - 1.0i * m_Omega * _widthOmega(q2) ) ) *
```

Comment:
``Omega -> omega`` every please. Function and variable identifies are ``lower_case_with_underscores``, not ``camelCase``.